dagobah/core/dagobah.py

```diff
@@ -245,7 +245,8 @@ def add_task_to_job(self, job_or_job_name, task_command, task_name=None,
         job.add_task(task_command, task_name, **kwargs)
         job.commit()
 
-    def add_jobtask_to_job(self, job_or_job_name, target_job, task_name=None):
+    def add_jobtask_to_job(self, job_or_job_name, target_job, task_name=None,
+                           **kwargs):
```

Comment:
just to explain this, the API passes a bunch of unused parameters, and also this matches it's related `add_task_to_job` signature better